api/v1alpha1/kubevirtcluster_types.go

```diff
@@ -85,6 +91,26 @@ type SSHKeys struct {
 	DataSecretName *string `json:"dataSecretName,omitempty"`
 }
 
+// LoadBalancerServiceTemplate describes the template for the cluster load balancer service.
+type LoadBalancerServiceTemplate struct {
```

Comment:
It may make sense to explicitly state what exactly gets routed over this service. Like: apiserver traffic, ingress, ...